addons/web/static/src/js/views/form/form_renderer.js

```diff
@@ -1058,6 +1058,10 @@ var FormRenderer = BasicRenderer.extend({
      */
     _onNavigationMove: function (ev) {
         ev.stopPropagation();
+        if (ev.data.originalEvent && ['next', 'previous'].includes(ev.data.direction)) {
```

Comment:
A little comment as to explain why would be welcome (because I don't know why :D)